README.md

```diff
@@ -65,6 +65,51 @@ High performance and convenient features are the main goals.
     cargo install --locked --git https://github.com/pamburus/hl.git
     ```
 
+* Run using [nix](https://nixos.org/download/)
+
+  ```sh
+  nix run github:pamburus/hl
+  ```
+
+  or with [nix profile](https://nix.dev/manual/nix/2.22/command-ref/new-cli/nix3-profile-install):
+
+  ```sh
+  nix profile install github:pamburus/hl
+  ```
+
+* Install the package by using [nix-flakes](https://wiki.nixos.org/wiki/Flakes)
+
+<details>
```

Comment:
Please indent the whole paragraph with two space under this section (I mean nix-flakes).
It should align properly then.